backend/src/Designer/Services/Implementation/SchemaModelService.cs

```diff
@@ -431,5 +435,20 @@ private bool NamespaceNeedsToBeSeparated(ApplicationMetadata application,
         {
             return application.DataTypes.All(d => d.AppLogic?.ClassRef != $"Altinn.App.Models.{csharpModelName}");
         }
+
+        public async Task<DataType> GetModelMetadata(string org, string app, string modelName)
+        {
+            ApplicationMetadata applicationMetadata = await _applicationMetadataService.GetApplicationMetadataFromRepository(org, app);
+            DataType dataType = applicationMetadata.DataTypes.Find((dataType) => dataType.Id == modelName);
+            return dataType;
+        }
+
+        public async Task UpdateModelMetadata(string org, string app, string modelName, DataType dataType)
```

Comment:
Is a better name of this method `DeleteDataTypeFromAppMetadata` or something similar?